lib/restforce_mock/sandbox.rb

```diff
@@ -1,13 +1,19 @@
 module RestforceMock
   module Sandbox
 
-    def self.add_object(name, id, values)
+    def self.add_object(name, id, values, params = {})
```

Comment:
Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used.